NeoMathEngine/src/MathEngineCommon.h

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 
 #include <cassert>
 #include <new>
+#include <stdexcept>
```

Comment:
Why here?

Reply:
I've met troubles when inlcluded this file in CudaDevice.h.
And I put `#include <stdexcept>` here because the macro defined in this file has `std::logic_error` in its code.

Reply:
What kind of problems? Is it still actual after the change of CudaDevice.h?